assets/src/edit-story/components/inspector/document/status.js

```diff
@@ -49,12 +49,17 @@ function StatusPanel() {
     state: { statuses },
   } = useInspector();
 
-  const {
-    state: {
-      story: { status, password },
-    },
-    actions: { updateStory, deleteStory },
-  } = useStory();
+  const { status, password, updateStory, deleteStory } = useStory((state) => {
```

Comment:
Eslint makes things really hard here, but I'm thinking if we addressed that if the following form would work well:

```
  const { status, password, updateStory, deleteStory } = useStory(({
      state: { story: { status, password } },
      actions: { updateStory, deleteStory },
  }) => ({status, password, updateStory, deleteStory}));
```

Reply:
Much nicer :)
Updated.